tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -503,3 +506,29 @@ async def test_train_persist_load_with_composite_entities(
     assert loaded.pipeline
     text = "I am looking for an italian restaurant"
     assert loaded.parse(text) == trained.parse(text)
+
+
+async def test_process_gives_diagnostic_data(
+    component_builder: ComponentBuilder, trained_nlu_moodbot_path: Text,
```

Comment:
`component_builder` seems unused?

Reply:
It is used by `Interpreter.load(..., component_builder)`

Reply:
but that one has a default for it so it's not needed, is it?

Reply:
I guess so. Its just that all other tests do that and I don't know why, so I copied this over. Will remove it now.